process/database/elasticSearchConnector.go

```diff
@@ -151,6 +150,32 @@ func (esc *elasticSearchConnector) doSearchRequest(query object, index string, s
 	return decodedBody, nil
 }
 
+func (esc *elasticSearchConnector) doSearchRequestTx(address string, index string, size int) (object, error) {
+	res, err := esc.client.Search(
+		esc.client.Search.WithIndex(index),
+		esc.client.Search.WithSize(size),
+		esc.client.Search.WithQuery("sender%"+address+"+receiver%"+address),
```

Comment:
Do you think it is possible to use this approach?

```
q := u.Query()
q.Set("sender", "...")
u.RawQuery = q.Encode()
```

https://stackoverflow.com/a/26987017/1475331